tests/tests.rs

```diff
@@ -38,12 +39,6 @@ fn run_cbindgen(
         command.arg("--style").arg(style_str(style));
     }
 
-    command.arg("-o").arg(output);
-
-    if env::var("CBINDGEN_TEST_VERIFY").is_ok() {
-        command.arg("--verify");
```

Comment:
It's a reasonable thing to use on CI if cbindgen is used offline rather than from a build script, I'd keep it.
(Also it's a public cbindgen's interface after all, someone else is probably using it.)